apps/librelingo_json_export/pyproject.toml

```diff
@@ -18,7 +18,7 @@ editdistance = "^0.5.3"
 pytest = "^6.2.2"
 pytest-watch = "^4.2.0"
 pyfakefs = "^4.3.3"
-librelingo-fakes = "^3.0.0"
+librelingo-fakes = {path = "../librelingo_fakes", develop = false}
```

Comment:
For me anyway, this seems to make testing a lot easier. If this is bad in some way, I can revert.

Reply:
Good idea!